### ISSUE TITLE
Sharing JSON

### ISSUE THREAD

**Issue by ultormagister** [KasemJaffer/receive_sharing_intent#287]
In ios, after following the set up suggested in the readme, it does not seem possible to process the shared json files. When sharing, the app is correctly found among those available for sharing but the SharedMediaFile array remains empty. This does not happen for other file formats.

**Comment by caiorondon** [comment]
+1, haven't been able to make it work as well with PDF files.

**Comment by ultormagister** [comment]
@caiorondon in my case pdf works well, at least they are processed and available in the SharedMediaFile array using the last version of the package (1.6.8)

**Comment by caiorondon** [comment]
> @caiorondon in my case pdf works well, at least they are processed and available in the SharedMediaFile array using the last version of the package (1.6.8)

Yeah, I made it work. I might have missed something stupid in the docs.
